### DISCUSSION TITLE
There is no Vercel account linked with this GitHub account.

### DISCUSSION THREAD

**Discussion by killianhuyghe** [vercel/community#4138] [Account Issue] [answered]
### Question

Hello,

I can't sign in to my Vercel account since at least 4 days ago (I get the error in the title). I have sent an email to `registration@vercel.com` but, because I can't sign in, it seems my case is not handled with the correct priority (the email I got back says I should sign in to access the Pro plan support 🤦‍♂️).

This is very problematic as I can't, for example, look at the function logs for debugging. Is there any way to speed up the process?

My case number is `00158274`.

Thank you for your help.

**Comment by amyegan** [comment, marked as answer]
Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the earlier cases that were opened before they reach yours. We all appreciate your patience and I'm glad to see you are no longer waiting on a response. 

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

Because there has been some more activity on your case since your original post here, I'm going to close this discussion.

**Reply by killianhuyghe** [reply]
Sure, I understand that, I was just surprised as I usually have a response from the support in less than 24h so I assumed the extra wait was due to having the wrong priority on the ticket. Anyway, all solved now, thank you.
